tests/tests_lite/plugins/collectives/test_torch_collective.py

```diff
@@ -10,9 +11,7 @@
 from lightning_lite.plugins.environments import LightningEnvironment
 from lightning_lite.strategies import DDPStrategy
 from lightning_lite.strategies.launchers.multiprocessing import _MultiProcessingLauncher
-from lightning_lite.utilities.imports import _TORCH_GREATER_EQUAL_1_11, _TORCH_GREATER_EQUAL_1_12
-
-torch_test_assert_close = torch.testing.assert_close if _TORCH_GREATER_EQUAL_1_12 else torch.testing.assert_allclose
```

Comment:
@otaj why did you remove this? isn't this needed to support <1.12?

Reply:
Nope, `torch.testing.assert_close` is available since 1.9 https://github.com/pytorch/pytorch/issues/61844